src/foam/util/filesystem/doc.flow

```diff
@@ -0,0 +1,3 @@
+<title>Package foam.util.filesystem</title>
+
+This package contains utilities for interacting with the filesystem.
```

Comment:
Could add a little more here if you don't mind. Even pasting the description of this PR in could be helpful.

Reply:
This is inside `foam/util/filesystem` where I put the file walker, so I'll add a new flow doc in `foam/foamlink` with that description

Reply:
I added earlier but in the wrong branch - should be there now